docs/introduction/background.md

```diff
@@ -4,28 +4,36 @@ Background
-- A **detector**, which records the mass-to-charge ratio of each ion against their relative abundance in a plot known as a {term}`mass spectrum`.
+- An **ion source**, which generates ions from the incoming sample. In LC-MS, the sample elutes from the LC column at different retentions times.
+- A **mass analyzer**, which separates the ions according to their mass-to-charge ratio. There are several types such as time of flight (TOF) and quadrupole mass analyzers. Depending on the mass analyzer, OpenMS offers calibration tools, so that highly accurate results can be achieved.
+- A **detector**, which records the mass-to-charge ratio of each ion against their relative abundance in a plot known as a mass spectrum.
 
 ![mass spectrometry](../images/introduction/introduction_MS.png)
 
+Sometimes a specialized form of mass spectrometry called Tandem Mass Spectrometry ({term}`MS/MS` spectrometry) is used when better identification is required. With MS/MS spectrometry,  an ion is isolated, fragmented using an inert gas and a second mass spectrum is recorded from the ion fragments. In this context, the primary ion is called the precursor ion and the spectrum from the fragments is called an  MS2 ({term}`MS/MS`) spectrum, which can help to resolve the ambiguities in identification of the precursor ion. Tandem mass spectrometry is especially useful for linear polymers like proteins, RNA and DNA and the fragments typically break the polymer into two parts. For example, peptides (short strands of amino acids, part of a protein) typically break between each of the amino acids, leading to a so-called ion ladder where the distance between each peak in the MS2 spectrum reveals the identity of the amino acid, as most amino acids have different masses. 
+
 ## Liquid Chromatography - Mass Spectrometry
-Liquid chromatography is often coupled with mass spectrometry to reduce complexity in the mass spectra. The separated analytes from the liquid chromatography setup are directly injected into the ion source from the mass spectrometry setup. Multiple analytes that elute at the same time are separated by mass using the mass spectrometer.
+
+Liquid chromatography is often coupled with mass spectrometry to reduce complexity in the mass spectra. If complex samples were directly fed to a mass spectrometer, you would not be able to detect the less abundant analyte ions. The separated analytes from the liquid chromatography setup are directly injected into the ion source from the mass spectrometry setup. Multiple analytes that elute at the same time are separated by mass using the mass spectrometer. 
 
 ![lc-ms setup](../images/introduction/hplc-mass-spectrometry-setup.png)
+
+From the LC-MS setup, a set of spectra called a peak map is produced. In a peak map, each spectrum represents an analyte eluted at a particular retention time. 
```

Comment:
A spectrum can contain peaks from multiple different analytes. Perhaps add that each peak has a retention time, mass-to-charge and intensity dimension.